.github/workflows/functional_all_db.yml

```diff
@@ -111,26 +105,23 @@ jobs:
     container:
       image: cypress/included:9.6.1
     services:
-      mariadb:
-        image: mariadb:latest
+      mysql:
```

Comment:
Maybe `mysql:latest`?

Reply:
No, I think it's bad to have the version flow automatically as it may break the tests, and we would not test anymore or don't know because it's a new version. It's better to create a PR to change/add a version when it's available and then merge when it works. I will bet you agree since it's the same problem as with using `nameof()` of column names.

Reply:
But, if MySQL changed their collation next time, it would be nice to target latest so we are aware of an issue before we bring it to main. Either way, it's not a big deal

Reply:
> But, if MySQL changed their collation next time

Yes, we need to be proactive. We are already with all our dependencies, like we are with dotnet.